.github/workflows/helm_charts_scalar.yml

```diff
@@ -76,8 +76,10 @@ jobs:
         run: |
           # TODO If more charts are supported by kubeaudit, they should be added.
           # Change to `ls charts` when all charts are supported.
-          chart_dirs=(envoy scalardb)
+          chart_dirs=(envoy scalardb scalardl scalardl-audit)
           for chart_dir in ${chart_dirs[@]}; do
+            echo "helm dependency build charts/${chart_dir} chart..."
+            helm dependency build charts/${chart_dir}
```

Comment:
Add the `helm dependency` command. We need to run it before checking by kubeaudit.
This is a bug fix.

After this fixing, CI doesn't return an error as follows.
https://github.com/scalar-labs/helm-charts/runs/8029109027?check_suite_focus=true#step:6:27